shared/common-adapters/icon.native.js

```diff
@@ -33,19 +34,18 @@ class Icon extends Component<void, Exact<Props>, void> {
     const backgroundColor = this.props.style && {backgroundColor: this.props.style.backgroundColor} || {}
 
     // Extract style props for our container and box
-    let containerProps = {
-      ...this.props.style,
-      bottom: undefined,
-      color: undefined,
-      fontSize: undefined,
-      height: undefined,
-      left: undefined,
-      position: undefined,
-      right: undefined,
-      textAlign: undefined,
-      top: undefined,
-      width: undefined,
-    }
+    let containerProps = omit(this.props.style, [
```

Comment:
Stylistically I think it would be nicer to do:

```
const {bottom, height, left, etc, ...containerProps} = this.props.style || {}
const clickableBoxStyle = {bottom, height, left, etc}
```

You could skip the `lodash` import and all the repetitions of this.props.style below.